src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -8,6 +8,10 @@ public class Bulldozer extends Machine {
     public Bulldozer() {
     }
 
```

Comment:
Due to the task, should we add some type-specific fields to these classes? (Bulldozer, Excavator, Truck)

Reply:
we shoud feel free to add specific fiels / i guess its not important